frontend/src/__tests__/integration/pages/projects/ProjectView.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { test, expect } from '@playwright/test';
+
+test('Create project', async ({ page }) => {
+  await page.goto(
+    './iframe.html?id=tests-integration-pages-projects-projectview--create-project&viewMode=story',
+  );
+
+  // wait for page to load
+  await page.waitForSelector('text=Create data science project');
+
+  // Test that can submit on valid form
+  await page.getByLabel('Name *', { exact: true }).click();
+  await page.getByLabel('Name *', { exact: true }).fill('Test Project');
+  await page.getByLabel('Description').click();
+  await page.getByLabel('Description').fill('Test project description');
+  await expect(page.getByRole('button', { name: 'Create' })).toBeEnabled();
+
+  // Test invalid resource name
+  await page.getByLabel('Resource name *').fill('test project');
+  await expect(page.getByRole('button', { name: 'Create' })).toBeDisabled();
+
+  await page.getByLabel('Resource name *').fill('test_project');
+  await expect(page.getByRole('button', { name: 'Create' })).toBeDisabled();
+
+  await page.getByLabel('Resource name *').fill('-test-project-');
+  await expect(page.getByRole('button', { name: 'Create' })).toBeDisabled();
+
+  await page.getByLabel('Resource name *').fill('123test-project123');
+  await expect(page.getByRole('button', { name: 'Create' })).toBeEnabled();
+
+  await page.getByLabel('Resource name *').fill('%^&test-project@#$');
+  await expect(page.getByRole('button', { name: 'Create' })).toBeDisabled();
+
+  await page.getByLabel('Resource name *').fill('test.project');
+  await expect(page.getByRole('button', { name: 'Create' })).toBeDisabled();
+
+  await page.getByLabel('Resource name *').fill('test-project');
+  await expect(page.getByRole('button', { name: 'Create' })).toBeEnabled();
```

Comment:
Am I missing something -- this does not appear to be part of the tests:
![image](https://github.com/opendatahub-io/odh-dashboard/assets/8126518/b618693e-44b7-4840-9490-52b253c94ecd)

Reply:
I outline this in the testing doc.

Storybook now only provides the mocking and setup for tests. The play function will either do nothing or get to the correct view. (this is for cases where we cant directly navigate to a view like a delete modal)

playwright is now used on top of these stories and implements all the tests. Playwright tests dont show up in the storybook UI, but instead are run by calling `npm run test:integration`

Reply:
Hmmm... okay. I need to read the doc better, I skimmed and this caused me confusion I bet.

Reply:
i will go over the doc again and remove some fluff to make things more clear

Reply:
updated the doc. should be more of a step by step guide now